docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -0,0 +1,128 @@
+[id="verifying-the-image-service-adoption_{context}"]
+
+= Verifying the {image_service} adoption
+
+Verify that you successfully adopted your {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
+//kgilliga: Does this procedure apply to all backends, or do some verification steps only apply to specific backends?
+
+.Procedure
+
+. Test the glance service from the {rhos_prev_long} CLI. You can compare and make sure the configuration has been correctly applied to the glance pods:
++
+----
+./os-diff cdiff --service glance -c /etc/glance/glance.conf.d/02-config.conf  -o glance_patch.yaml --frompod -p glance-api
+----
++
+If no line appears, then the configuration is correctly done.
+
+. Inspect the resulting glance pods and ensure that Ceph secrets are properly mounted:
```

Comment:
I think this line (L18) should be moved to L34, where we actually check the Ceph bits, and yeah, checking the Ceph secret is an action you won't do for other backends.